frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/CollapsibleHeader/CollapsibleHeader.module.css

```diff
@@ -0,0 +1,53 @@
+.header {
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--spacing-2);
+  border-bottom: 1px solid var(--global-border);
+  background-color: var(--pane-background);
+  cursor: pointer;
+}
+
+@media screen and (max-width: 767px) {
+  .header {
+    position: sticky;
+  }
+  .header[data-state='closed'] {
+    position: relative;
+    top: auto;
+  }
+}
+
+.iconTitleContainer {
+  display: grid;
+  grid-auto-flow: column;
+  gap: var(--spacing-1);
+  align-items: center;
+  color: var(--overlay-70);
+}
+
+.heading {
+  color: var(--global-foreground);
+  font-size: var(--font-size-3);
+  font-style: normal;
+  line-height: normal;
+}
+
+.iconContainer {
+  display: grid;
+  grid-auto-flow: column;
+  gap: var(--spacing-1);
+  align-items: center;
+}
+
+.content.visible {
+  max-height: max-content;
+  overflow: hidden;
+  transition: max-height 0.1s var(animation-timing-function);
```

Comment:
Animation seems to have stopped working. Could you please check?

https://github.com/user-attachments/assets/be1d37b6-6b69-4403-ab7a-cb576677fdb4